tests/eas/preliminary.py

```diff
@@ -188,6 +190,66 @@ def test_energy_model_present(self):
                 '- Kernel built without (CONFIG_SCHED_DEBUG && CONFIG_SYSCTL)\n'
                 '- No energy model in kernel')
 
+class TestEnergyModelSanity(BasicCheckTest):
```

Comment:
I don't think this test should live in `tests/eas/preliminary.py`. Some platforms will fail those tests but that doesn't mean EAS won't work on them. The semantic for `prelminary.py` is: if one of the tests fail, go fix your platform before running anything else because something is broken. Your test is somewhat different, so I'd prefer to to have it in a new file called `tests/eas/energy_model.py` or something like that. That said, others might disagree. @credp @derkling : any thoughts ?

Reply:
Done --> the tests has been moved into tests/eas/energy_model.py